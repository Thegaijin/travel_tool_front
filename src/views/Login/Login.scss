.login-page {
  &__andela-logo {
    padding: 80px;
  }
  
  &__landing-page-map {
  width: 108%;
  }
  
  &__travel-request-text {
    padding: 80px;
    padding-top: 59px;
    height: 102px;
    width: 292px;
    color: #3359DB;
    font-family: "DIN Pro";
    font-size: 40px;
    font-weight: 600;
    line-height: 51px;
    font-style: normal;
    margin-bottom: 0px;
    padding-bottom: 55px;
  }
  
  &__login-btn {
    padding: 0%;
    height: 50px;
    width: 252px;
    border-radius: 4px;
    background-color: #3359DB !important;
    margin-left: 80px;
    text-transform: none;
  }
  
  &__login-to-get-started-text {
    height: 41px;
    width: 162px;
    color: #FFFFFF;
    font-family: "DIN Pro";
    font-size: 18px;
    font-weight: 600;
    line-height: 40px;
  }
  
  &__how-to-book-a-trip-text {
    height: 21px;
    width: 129px;
    color: #3359DB;
    font-family: "DIN Pro";
    font-size: 16px;
    line-height: 21px;
    margin-left: 12.12px;
  }
  
  &__andela-travel-policy {
    height: 21px;
    width: 139px;
    color: #3359DB;
    font-family: "DIN Pro";
    font-size: 16px;
    line-height: 21px;
    margin-left: 12.12px;
  }
  
  &__how-to-book-a-trip-link {
    padding: 80px;
    padding-top: 215px;
    padding-bottom: 0px;
    margin-bottom: 10px;
  }
  
  &__andela-travel-policy-link {
    padding-left: 80px;
    padding-right: 90px;
    margin-bottom: 20px;
    margin-bottom: 0px;
  }
  &__google-white {
    margin-top: -7px;
    margin-right: 19px;
  }
}






