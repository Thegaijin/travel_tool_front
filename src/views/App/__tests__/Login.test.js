import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../../Login/Login';

it('should render Login page correctly', () => {
const tree = renderer.create(<Login />).toJSON();
expect(tree).toMatchSnapshot();
});