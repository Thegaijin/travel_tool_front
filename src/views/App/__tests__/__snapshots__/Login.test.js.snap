// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render Login page correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login
    history={
        Object {
            "push": [Function],
          }
    }
    isAuthenticated={true}
    verifyUser={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="mdl-layout__content"
>
        <div
                className="mdl-grid mdl-grid--no-spacing"
        >
                <div
                        className="mdl-cell mdl-cell--5-col"
                >
                        <img
                                alt="Andela Logo"
                                className="login-page__andela-logo"
                                src="andela-logo.svg"
                        />
                        <p
                                className="login-page__travel-request-text"
                        >
                                Travel Requests Made Easier
                        </p>
                        <button
                                className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                                onClick={[Function]}
                                type="button"
                        >
                                <img
                                        alt="Google Symbol"
                                        className="login-page__google-white"
                                        src="Google-white.svg"
                                />
                                <span
                                        className="login-page__login-to-get-started-text"
                                >
                                        Login to Get Started
                                </span>
                        </button>
                        <TextLink
                                altText="Video Symbol"
                                imageSrc="video.svg"
                                text="How to book a trip"
                                textClass="login-page__how-to-book-a-trip-text"
                                textLinkClass="login-page__how-to-book-a-trip-link"
                        />
                        <TextLink
                                altText="File Symbol"
                                imageSrc="file.svg"
                                text="Andela travel policy"
                                textClass="login-page__andela-travel-policy"
                                textLinkClass="login-page__andela-travel-policy-link"
                        />
                </div>
                <div
                        className="mdl-cell mdl-cell--7-col"
                >
                        <img
                                alt="Road map"
                                className="login-page__landing-page-map"
                                src="cover.svg"
                        />
                </div>
        </div>
</div>,
      "className": "mdl-layout mdl-js-layout login-page",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="mdl-grid mdl-grid--no-spacing"
>
          <div
                    className="mdl-cell mdl-cell--5-col"
          >
                    <img
                              alt="Andela Logo"
                              className="login-page__andela-logo"
                              src="andela-logo.svg"
                    />
                    <p
                              className="login-page__travel-request-text"
                    >
                              Travel Requests Made Easier
                    </p>
                    <button
                              className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                              onClick={[Function]}
                              type="button"
                    >
                              <img
                                        alt="Google Symbol"
                                        className="login-page__google-white"
                                        src="Google-white.svg"
                              />
                              <span
                                        className="login-page__login-to-get-started-text"
                              >
                                        Login to Get Started
                              </span>
                    </button>
                    <TextLink
                              altText="Video Symbol"
                              imageSrc="video.svg"
                              text="How to book a trip"
                              textClass="login-page__how-to-book-a-trip-text"
                              textLinkClass="login-page__how-to-book-a-trip-link"
                    />
                    <TextLink
                              altText="File Symbol"
                              imageSrc="file.svg"
                              text="Andela travel policy"
                              textClass="login-page__andela-travel-policy"
                              textLinkClass="login-page__andela-travel-policy-link"
                    />
          </div>
          <div
                    className="mdl-cell mdl-cell--7-col"
          >
                    <img
                              alt="Road map"
                              className="login-page__landing-page-map"
                              src="cover.svg"
                    />
          </div>
</div>,
        "className": "mdl-layout__content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="mdl-cell mdl-cell--5-col"
>
              <img
                            alt="Andela Logo"
                            className="login-page__andela-logo"
                            src="andela-logo.svg"
              />
              <p
                            className="login-page__travel-request-text"
              >
                            Travel Requests Made Easier
              </p>
              <button
                            className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                            onClick={[Function]}
                            type="button"
              >
                            <img
                                          alt="Google Symbol"
                                          className="login-page__google-white"
                                          src="Google-white.svg"
                            />
                            <span
                                          className="login-page__login-to-get-started-text"
                            >
                                          Login to Get Started
                            </span>
              </button>
              <TextLink
                            altText="Video Symbol"
                            imageSrc="video.svg"
                            text="How to book a trip"
                            textClass="login-page__how-to-book-a-trip-text"
                            textLinkClass="login-page__how-to-book-a-trip-link"
              />
              <TextLink
                            altText="File Symbol"
                            imageSrc="file.svg"
                            text="Andela travel policy"
                            textClass="login-page__andela-travel-policy"
                            textLinkClass="login-page__andela-travel-policy-link"
              />
</div>,
            <div
              className="mdl-cell mdl-cell--7-col"
>
              <img
                            alt="Road map"
                            className="login-page__landing-page-map"
                            src="cover.svg"
              />
</div>,
          ],
          "className": "mdl-grid mdl-grid--no-spacing",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="Andela Logo"
                  className="login-page__andela-logo"
                  src="andela-logo.svg"
/>,
                <p
                  className="login-page__travel-request-text"
>
                  Travel Requests Made Easier
</p>,
                <button
                  className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                  onClick={[Function]}
                  type="button"
>
                  <img
                                    alt="Google Symbol"
                                    className="login-page__google-white"
                                    src="Google-white.svg"
                  />
                  <span
                                    className="login-page__login-to-get-started-text"
                  >
                                    Login to Get Started
                  </span>
</button>,
                <TextLink
                  altText="Video Symbol"
                  imageSrc="video.svg"
                  text="How to book a trip"
                  textClass="login-page__how-to-book-a-trip-text"
                  textLinkClass="login-page__how-to-book-a-trip-link"
/>,
                <TextLink
                  altText="File Symbol"
                  imageSrc="file.svg"
                  text="Andela travel policy"
                  textClass="login-page__andela-travel-policy"
                  textLinkClass="login-page__andela-travel-policy-link"
/>,
              ],
              "className": "mdl-cell mdl-cell--5-col",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Andela Logo",
                  "className": "login-page__andela-logo",
                  "src": "andela-logo.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Travel Requests Made Easier",
                  "className": "login-page__travel-request-text",
                },
                "ref": null,
                "rendered": "Travel Requests Made Easier",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Google Symbol"
                      className="login-page__google-white"
                      src="Google-white.svg"
/>,
                    <span
                      className="login-page__login-to-get-started-text"
>
                      Login to Get Started
</span>,
                  ],
                  "className": "mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn",
                  "onClick": [Function],
                  "type": "button",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Google Symbol",
                      "className": "login-page__google-white",
                      "src": "Google-white.svg",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Login to Get Started",
                      "className": "login-page__login-to-get-started-text",
                    },
                    "ref": null,
                    "rendered": "Login to Get Started",
                    "type": "span",
                  },
                ],
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "altText": "Video Symbol",
                  "imageSrc": "video.svg",
                  "text": "How to book a trip",
                  "textClass": "login-page__how-to-book-a-trip-text",
                  "textLinkClass": "login-page__how-to-book-a-trip-link",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "altText": "File Symbol",
                  "imageSrc": "file.svg",
                  "text": "Andela travel policy",
                  "textClass": "login-page__andela-travel-policy",
                  "textLinkClass": "login-page__andela-travel-policy-link",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Road map"
                className="login-page__landing-page-map"
                src="cover.svg"
/>,
              "className": "mdl-cell mdl-cell--7-col",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Road map",
                "className": "login-page__landing-page-map",
                "src": "cover.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="mdl-layout__content"
>
          <div
                    className="mdl-grid mdl-grid--no-spacing"
          >
                    <div
                              className="mdl-cell mdl-cell--5-col"
                    >
                              <img
                                        alt="Andela Logo"
                                        className="login-page__andela-logo"
                                        src="andela-logo.svg"
                              />
                              <p
                                        className="login-page__travel-request-text"
                              >
                                        Travel Requests Made Easier
                              </p>
                              <button
                                        className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                                        onClick={[Function]}
                                        type="button"
                              >
                                        <img
                                                  alt="Google Symbol"
                                                  className="login-page__google-white"
                                                  src="Google-white.svg"
                                        />
                                        <span
                                                  className="login-page__login-to-get-started-text"
                                        >
                                                  Login to Get Started
                                        </span>
                              </button>
                              <TextLink
                                        altText="Video Symbol"
                                        imageSrc="video.svg"
                                        text="How to book a trip"
                                        textClass="login-page__how-to-book-a-trip-text"
                                        textLinkClass="login-page__how-to-book-a-trip-link"
                              />
                              <TextLink
                                        altText="File Symbol"
                                        imageSrc="file.svg"
                                        text="Andela travel policy"
                                        textClass="login-page__andela-travel-policy"
                                        textLinkClass="login-page__andela-travel-policy-link"
                              />
                    </div>
                    <div
                              className="mdl-cell mdl-cell--7-col"
                    >
                              <img
                                        alt="Road map"
                                        className="login-page__landing-page-map"
                                        src="cover.svg"
                              />
                    </div>
          </div>
</div>,
        "className": "mdl-layout mdl-js-layout login-page",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="mdl-grid mdl-grid--no-spacing"
>
            <div
                        className="mdl-cell mdl-cell--5-col"
            >
                        <img
                                    alt="Andela Logo"
                                    className="login-page__andela-logo"
                                    src="andela-logo.svg"
                        />
                        <p
                                    className="login-page__travel-request-text"
                        >
                                    Travel Requests Made Easier
                        </p>
                        <button
                                    className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                                    onClick={[Function]}
                                    type="button"
                        >
                                    <img
                                                alt="Google Symbol"
                                                className="login-page__google-white"
                                                src="Google-white.svg"
                                    />
                                    <span
                                                className="login-page__login-to-get-started-text"
                                    >
                                                Login to Get Started
                                    </span>
                        </button>
                        <TextLink
                                    altText="Video Symbol"
                                    imageSrc="video.svg"
                                    text="How to book a trip"
                                    textClass="login-page__how-to-book-a-trip-text"
                                    textLinkClass="login-page__how-to-book-a-trip-link"
                        />
                        <TextLink
                                    altText="File Symbol"
                                    imageSrc="file.svg"
                                    text="Andela travel policy"
                                    textClass="login-page__andela-travel-policy"
                                    textLinkClass="login-page__andela-travel-policy-link"
                        />
            </div>
            <div
                        className="mdl-cell mdl-cell--7-col"
            >
                        <img
                                    alt="Road map"
                                    className="login-page__landing-page-map"
                                    src="cover.svg"
                        />
            </div>
</div>,
          "className": "mdl-layout__content",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="mdl-cell mdl-cell--5-col"
>
                <img
                                alt="Andela Logo"
                                className="login-page__andela-logo"
                                src="andela-logo.svg"
                />
                <p
                                className="login-page__travel-request-text"
                >
                                Travel Requests Made Easier
                </p>
                <button
                                className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                                onClick={[Function]}
                                type="button"
                >
                                <img
                                                alt="Google Symbol"
                                                className="login-page__google-white"
                                                src="Google-white.svg"
                                />
                                <span
                                                className="login-page__login-to-get-started-text"
                                >
                                                Login to Get Started
                                </span>
                </button>
                <TextLink
                                altText="Video Symbol"
                                imageSrc="video.svg"
                                text="How to book a trip"
                                textClass="login-page__how-to-book-a-trip-text"
                                textLinkClass="login-page__how-to-book-a-trip-link"
                />
                <TextLink
                                altText="File Symbol"
                                imageSrc="file.svg"
                                text="Andela travel policy"
                                textClass="login-page__andela-travel-policy"
                                textLinkClass="login-page__andela-travel-policy-link"
                />
</div>,
              <div
                className="mdl-cell mdl-cell--7-col"
>
                <img
                                alt="Road map"
                                className="login-page__landing-page-map"
                                src="cover.svg"
                />
</div>,
            ],
            "className": "mdl-grid mdl-grid--no-spacing",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Andela Logo"
                    className="login-page__andela-logo"
                    src="andela-logo.svg"
/>,
                  <p
                    className="login-page__travel-request-text"
>
                    Travel Requests Made Easier
</p>,
                  <button
                    className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn"
                    onClick={[Function]}
                    type="button"
>
                    <img
                                        alt="Google Symbol"
                                        className="login-page__google-white"
                                        src="Google-white.svg"
                    />
                    <span
                                        className="login-page__login-to-get-started-text"
                    >
                                        Login to Get Started
                    </span>
</button>,
                  <TextLink
                    altText="Video Symbol"
                    imageSrc="video.svg"
                    text="How to book a trip"
                    textClass="login-page__how-to-book-a-trip-text"
                    textLinkClass="login-page__how-to-book-a-trip-link"
/>,
                  <TextLink
                    altText="File Symbol"
                    imageSrc="file.svg"
                    text="Andela travel policy"
                    textClass="login-page__andela-travel-policy"
                    textLinkClass="login-page__andela-travel-policy-link"
/>,
                ],
                "className": "mdl-cell mdl-cell--5-col",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Andela Logo",
                    "className": "login-page__andela-logo",
                    "src": "andela-logo.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Travel Requests Made Easier",
                    "className": "login-page__travel-request-text",
                  },
                  "ref": null,
                  "rendered": "Travel Requests Made Easier",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <img
                        alt="Google Symbol"
                        className="login-page__google-white"
                        src="Google-white.svg"
/>,
                      <span
                        className="login-page__login-to-get-started-text"
>
                        Login to Get Started
</span>,
                    ],
                    "className": "mdl-button mdl-js-button mdl-button--raised mdl-button--colored login-page__login-btn",
                    "onClick": [Function],
                    "type": "button",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "Google Symbol",
                        "className": "login-page__google-white",
                        "src": "Google-white.svg",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Login to Get Started",
                        "className": "login-page__login-to-get-started-text",
                      },
                      "ref": null,
                      "rendered": "Login to Get Started",
                      "type": "span",
                    },
                  ],
                  "type": "button",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "altText": "Video Symbol",
                    "imageSrc": "video.svg",
                    "text": "How to book a trip",
                    "textClass": "login-page__how-to-book-a-trip-text",
                    "textLinkClass": "login-page__how-to-book-a-trip-link",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "altText": "File Symbol",
                    "imageSrc": "file.svg",
                    "text": "Andela travel policy",
                    "textClass": "login-page__andela-travel-policy",
                    "textLinkClass": "login-page__andela-travel-policy-link",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Road map"
                  className="login-page__landing-page-map"
                  src="cover.svg"
/>,
                "className": "mdl-cell mdl-cell--7-col",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Road map",
                  "className": "login-page__landing-page-map",
                  "src": "cover.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
